admin/class-metabox.php

```diff
@@ -96,13 +92,8 @@ public static function translate_meta_boxes() {
 	 * @return  bool        Whether or not the meta box (and associated columns etc) should be hidden
 	 */
 	function is_metabox_hidden( $post_type = null ) {
-		if ( ! isset( $post_type ) ) {
-			if ( isset( $GLOBALS['post'] ) && ( is_object( $GLOBALS['post'] ) && isset( $GLOBALS['post']->post_type ) ) ) {
-				$post_type = $GLOBALS['post']->post_type;
-			}
-			elseif ( isset( $_GET['post_type'] ) && $_GET['post_type'] !== '' ) {
```

Comment:
I see it has been moved to https://github.com/Yoast/wordpress-seo/pull/2691/files#diff-2c885f2a1e799bc73174ed48861bc7f1R412